Lib/zoneinfo/__init__.py

```diff
@@ -0,0 +1,29 @@
+__all__ = [
+    "ZoneInfo",
+    "reset_tzpath",
+    "TZPATH",
+    "ZoneInfoNotFoundError",
+    "InvalidTZPathWarning",
+]
+
+from . import _tzpath
+from ._common import ZoneInfoNotFoundError
+
+try:
+    from _czoneinfo import ZoneInfo
```

Comment:
In the reference implementation, I had this as `zoneinfo._czoneinfo`, but when I tried to do that here, `make` was putting the compiled `.so` in `Modules/_czoneinfo`, and since `zoneinfo` has an `__init__`, `import zoneinfo._czoneinfo` did not work.

Does anyone know if there's a way to allow us to keep the C module as `zoneinfo._czoneinfo`?

Reply:
Not with the current build systems, unfortunately.

Reply:
:disappointed: Good to know. Thanks!